code/modules/clothing/modular_armor/attachments/modules.dm

```diff
@@ -58,6 +58,7 @@
 	desc = "Designed for mounting on modular armor. Providing a near immunity to being bathed in flames, and amazing flame retardant qualities, this is every pyromaniacs' first stop to survival. Will impact mobility."
 	icon_state = "mod_fire"
 	item_state = "mod_fire_a"
+	soft_armor = list("fire" = 200)
```

Comment:
```suggestion
	soft_armor = list(FIRE = 200)
```
use defines. Hard armour should be the same while you're at it.

Reply:
Done